### ISSUE TITLE
the internal `errno` module is documented in an unexpected location

### ISSUE THREAD

**Issue by jepler** [adafruit/circuitpython#7943]
it would be nice if it were documented.

**Comment by jepler** [comment]
It's actually documented at docs/library/errno.html but this is not obvious...

**Comment by dhalbert** [comment]
@tekktrik Yes, that makes sense. The shared-bindings list is generated by a script, right? Note that `errno` and other "library" modules are already accessible in the Module Support Matrix, so this might be a simple change to the script.

**Comment by tekktrik** [comment]
Sounds good!  I'll take this and make the relevant change!

**Comment by tekktrik** [comment]
It looks like this one (and others in this section) are could be moved over to the Core Modules section, but I'm not sure I could put it in order alphabetically.  Would it be work combining the "MicroPython libraries" section entirely into the "Core Modules" section?

**Comment by tekktrik** [comment]
https://github.com/adafruit/circuitpython/blob/main/shared-bindings/index.rst

**Comment by jepler** [comment]
@tekktrik are you planning to work on this? Otherwise, we'll just move it to long term.

**Comment by tekktrik** [comment]
I'm looking for clarification on my question above, but otherwise happy to complete it!

**Comment by dhalbert** [comment]
> It looks like this one (and others in this section) are could be moved over to the Core Modules section, but I'm not sure I could put it in order alphabetically. Would it be work combining the "MicroPython libraries" section entirely into the "Core Modules" section?

I think this would be OK, perhaps adding a warning at the top that though they appear to be in alphabetical order, some modules are at the bottom. Note that `help()` is already at the bottom. Could also add a link to the Module Support Matrix with a note that says that's another way in to the list.

We _could_ also list everything manually, but then every new module has to be added by hand, or else we have to automate that part.

There is also some way to add a hook routine that gets called that can re-sort. This may or may not help: https://stackoverflow.com/questions/8324959/how-can-i-reverse-order-the-toctree-provided-by-the-glob-flag-option

Note that alphabetization by title instead of filename has been proposed but not implemented:https://github.com/sphinx-doc/sphinx/issues/1430.

**Comment by dhalbert** [comment]
Fixed by #10160.
